force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -238,13 +306,15 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Set<String> createStringSet(List<String> inputList) {
         // Create a new Set of strings
-        Set<String> stringSet = new Set<String>();
+        Set<String> newOutputSet = new Set<String>();
 
         //add strings to the set
+        Set<String> newOutputSet = new Set<String>(inputList);
+        System.debug(newOutputSet);
 
-        return null; // Replace null with the variable you used to store the result
+        return newOutputSet; // Replace null with the variable you used to store the result
```

Comment:
good. You can be even more concise and skip the variable assignment. You don't really need it.  

```
 public static Set<String> createStringSet(List<String> inputList) {
        return new Set<String>(inputList); // Replace null with the variable you used to store the result
    }
    ```